### PR TITLE
Fix list external links when required

### PR DESCRIPTION
**Pull request by mario-kr**
Fixes the build error.
Review required, as I'm not too sure about side effects.
Thx in advance

### CODE REVIEW

File: imag-link/src/main.rs [hunk 1]
```diff
@@ -98,11 +98,9 @@ fn handle_internal_linking(rt: &Runtime) {
                     Ok(Some(e)) => {
                         e.get_internal_links()
                             .map(|iter| {
-                                if cmd.is_present("list-externals-too") {
-                                    iter.filter(|_| true)
-                                } else {
-                                    iter.filter(|id| !is_external_link_storeid(&id))
-                                }
+                                iter.filter(move |id| {
```
Comment (matthiasbeyer) [inline imag-link/src/main.rs]: I'm not sure we need the `move` here.

**Comment by matthiasbeyer** [comment]
I'm not sure why travis fails. I started a rebuild, lets see...

File: imag-link/src/main.rs (hunk 1 above)
Comment (mario-kr) [inline imag-link/src/main.rs]: I got a lifetime error w/o it. I guess I'm not yet a buddy of the borrow/lifetime checker 😞

**Comment by mario-kr** [comment]
> I'm not sure why travis fails. I started a rebuild, lets see...

Your branch wasnt rebased yet, so the older `Cargo.toml` is used, which doesnt specify a target. 1.11 doesnt really know workspaces, stable doesnt seem to care about a target specified, but it seems like beta (->the next version) requires a target.

If you rebase your branch, I can redo the changes.

**Comment by matthiasbeyer** [comment]
Rebased.

**Comment by mario-kr** [comment]
github conflict resolving is... fascinating.

**Comment by matthiasbeyer** [comment]
What do you mean?

**Comment by mario-kr** [comment]
It just never occured to me to resolve merge conflicts by pulling the branch i want to merge into. (->TIL)

**Comment by matthiasbeyer** [comment]
@mario-kr are you okay if I cherry-pick your commits from this branch to mine?

**Comment by mario-kr** [comment]
np

**Comment by matthiasbeyer** [comment]
Cherry-picked in https://github.com/matthiasbeyer/imag/pull/842/commits/ddd4a8b94a7edaf308e6e5a70baf65c12464f2d7

Thank you!

**Pull request closed without merging**